src/Features/Core/Portable/QuickInfo/QuickInfoOptions.cs

```diff
@@ -4,18 +4,17 @@
 
 using System.Runtime.Serialization;
 
-namespace Microsoft.CodeAnalysis.QuickInfo
+namespace Microsoft.CodeAnalysis.QuickInfo;
+
+[DataContract]
+internal readonly record struct QuickInfoOptions
 {
-    [DataContract]
-    internal readonly record struct QuickInfoOptions(
-        [property: DataMember(Order = 0)] bool ShowRemarksInQuickInfo = true,
-        [property: DataMember(Order = 1)] bool IncludeNavigationHintsInQuickInfo = true)
-    {
-        public QuickInfoOptions()
-            : this(ShowRemarksInQuickInfo: true)
-        {
-        }
+    [DataMember] public bool ShowRemarksInQuickInfo { get; init; } = true;
+    [DataMember] public bool IncludeNavigationHintsInQuickInfo { get; init; } = true;
```

Comment:
❔ Are we planning to make these required as well?

Reply:
No, these are not required - they have well defined default values,